spec/requests/api/querying_spec.rb

```diff
@@ -323,9 +323,21 @@ def create_vms_by_name(names)
 
       run_get vms_url, :expand => "resources", :attributes => "name,vendor"
 
-      expect_query_result(:vms, 1, 1)
```

Comment:
I prefer the latter because of the readability...I know exactly what's going on in the new test, and if I break something, it's much clearer in getting to a resolution to fixing it.

However, I understand wanting to encapsulate structure somewhere.  Perhaps there should be a separate test for structure versus content?